src/app/client/src/app/modules/dashboard/services/course-summary-courses/course-summary-courses.service.ts

```diff
@@ -0,0 +1,80 @@
+import { Injectable } from '@angular/core';
```

Comment:
@AnkitaMaheshChavan - create only one service and use for course and event also (both services functions are same def that's the reason create only one )